dev-lang/erlang/erlang-21.0.4.ebuild

```diff
@@ -184,12 +184,7 @@ src_install() {
 	dosym \
 		"${ERL_LIBDIR}/lib/erl_interface-${ERL_INTERFACE_VER}/bin/erl_call" \
 		/usr/bin/erl_call
-
-	if use smp; then
-		dosym "${ERL_LIBDIR}/erts-${ERL_ERTS_VER}/bin/beam.smp" /usr/bin/beam.smp
-	else
-		dosym "${ERL_LIBDIR}/erts-${ERL_ERTS_VER}/bin/beam" /usr/bin/beam
-	fi
+	dosym "${ERL_LIBDIR}/erts-${ERL_ERTS_VER}/bin/beam.smp" /usr/bin/beam.smp
```

Comment:
Use relative symlinks instead of absolute everywhere.

Reply:
done